modules/understanding-upgrade-channels.adoc

```diff
@@ -24,6 +24,7 @@ binary file for a specific version of {product-title} always installs that versi
 * `candidate-{product-version}`
```

Comment:
These types of changes that aren't related to Serverless docs should probably be a separate PR so that they can be reviewed by the proper docs SMEs?